packages/core/src/internal/decorators/property.ts

```diff
@@ -134,18 +134,18 @@ export interface PropertyDeclaration<Type = unknown, TypeHint = unknown> {
 }
 
 /**
```

Comment:
Lit renamed the `@internalProperty` decorator to `@state` to make it easier to spot internal state of a component. Here we follow the same convention but keep our specific defaults.